config.json

```diff
@@ -150,7 +150,7 @@
         "name": "Two Fer",
         "uuid": "7f49e997-4435-4f34-a020-bddc92c838ed",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "functions"],
```

Comment:
Probably should have default arguments (dunno what we called it in the big list).